src/main/java/org/xwiki/git/internal/DefaultGitManager.java

```diff
@@ -112,6 +102,23 @@ public Repository getRepository(String repositoryURI, String localDirectoryName,
         return repository;
     }
 
+    @Override
+    public Repository getRepository(String repositoryURI, String localDirectoryName)
+    {
+        // Simple CloneCommand with default settings
+        CloneCommand cloneCommand = Git.cloneRepository();
```

Comment:
Here we can do `createCloneCommand()` by adding it in the interface (so the `ScriptService` would also use it instead). I need opinions :)